src/test/scala/no/ndla/draftapi/integration/ConceptApiClientTest.scala

```diff
@@ -17,24 +17,33 @@ class ConceptApiClientTest extends UnitSuite with TestEnvironment {
     reset(conceptApiClient)
   }
 
-  test("Test that publishConceptsIfToPublishing publishes concepts with QUALITY_ASSURED status") {
+  test("Test that publishConceptsIfToPublishing publishes concepts with QUALITY_ASSURED or DRAFT status") {
```

Comment:
kanskje lurt å skrive test cases for alle statuser i `statusToPublish` lista? lettere å debugge hvis vi endrer på statusene i etterkant og testene feiler